### PR TITLE
Bump to version 0.1.1

### PR DESCRIPTION
**Pull request by Tbaut**
Is that all that's needed?

### CODE REVIEW

**Comment by amaurymartiny** [comment]
Missing some, the best is to do a CTRL+F for 0.1.0, and update all instances in all package.jsons

**Comment by amaurymartiny** [comment]
Also missing: the ones in the `dependencies` field in package.json.

After that, can you run a `yarn install` in the root folder? Then we're done.

**Comment by Tbaut** [comment]
Not sure about the lerna..

**Comment by amaurymartiny** [comment]
> Not sure about the lerna..

The lerna one should be kept 0.1.0

**Comment by Tbaut** [comment]
Thanks, the ctrl-f gave me so many things that I couldn't have a clear overview..

File: lerna.json [hunk 1]
```diff
@@ -3,5 +3,5 @@
   "npmClient": "yarn",
   "packages": ["packages/*"],
   "useWorkspaces": true,
-  "version": "0.1.0"
+  "version": "0.1.1"
```
Comment (amaurymartiny) [inline lerna.json]: 0.1.0 here.

**Pull request merged**